astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,424 @@
+
+```shell
+curl "https://laravel.build/fusionauth-quickstart-php-laravel-api?with=mariadb" | bash
+cd fusionauth-quickstart-php-laravel-api
+```
+
+### Initialize the Application
+
+Initialize the Laravel application using the following:
+
+```shell
+./vendor/bin/sail up
+```
+
+### Update Routes
+
+Change `routes/api.php` to add both `/make-change` and `/panic` endpoints for your ChangeBank application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/routes/api.php"} lang="php"/>
+
+### Add Security
+
+Include FusionAuth's library to add JWT support to the authentication system.
+
+```shell
+./vendor/bin/sail composer require fusionauth/jwt-auth-webtoken-provider web-token/jwt-signature-algorithm-rsa
+```
+
+### Create Fields in the User Entity
+
+When installing Laravel, it will automatically make a migration to create a `users` table in the database. You'll have to edit the file `database/migrations/2014_10_12_000000_create_users_table.php` to change the default behavior and use UUID for the primary key, remove the need for a password and the remember token.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/database/migrations/2014_10_12_000000_create_users_table.php"} lang="php"/>
+
+You'll also need to edit `app/Models/User.php` to make the database changes consistent with your model.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/Models/User.php"} lang="php"/>
+
+You may remove all other migrations, as you won't need them.
+
+```shell
+rm database/migrations/2014_10_12_100000_create_password_reset_tokens_table.php
+rm database/migrations/2019_08_19_000000_create_failed_jobs_table.php
+rm database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php
+```
+
+Run the remaining migrations to create the necessary tables in your database.
+
+```shell
+./vendor/bin/sail artisan migrate
+```
+
+### Provisioning New Users
+
+By default, Laravel only allows JWTs that correspond to users in your database, but one of the greatest benefits of using FusionAuth is to have a single source of truth of user management. So, you want your API to automatically provision new users when it receives a trusted JWT from FusionAuth.
+
+To do so, add a new [User Provider](https://laravel.com/docs/10.x/authentication#adding-custom-user-providers) at `app/FusionAuth/Providers/FusionAuthEloquentUserProvider.php` to create new users when receiving valid JWTs from FusionAuth.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/FusionAuth/Providers/FusionAuthEloquentUserProvider.php"} lang="php"/>
+
+Now, add an [Authentication Guard](https://laravel.com/docs/10.x/authentication#adding-custom-guards) in `app/FusionAuth/FusionAuthJWTGuard.php` to call that custom method from the User Provider created above.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/FusionAuth/FusionAuthJWTGuard.php"} lang="php"/>
```

Comment:
Not Found.  could be casing.  `JWT` here and `jwt` in complete-application
![Screenshot 2023-10-24 at 4 15 45 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/210a44a3-8aa6-4354-87ef-cf145d0e806c)